packages/squiggle-lang/benchmark/generators.ts

```diff
@@ -0,0 +1,48 @@
+export let generateFloatRange = (min: number, max: number): number =>
+  Math.random() * (max - min) + min;
+export let generateIntRange = (min: number, max: number): number =>
+  Math.floor(generateFloatRange(min, max));
+
+export let generateIntMin = (min: number): number => generateIntRange(min, 100);
+
+export let generateInt = (): number => generateIntMin(-100);
+
+let generatePositiveInt = (): number => generateIntMin(1);
+
+export let generateNormal = (): string =>
+  `normal(${generateInt()}, ${generatePositiveInt()})`;
+
+export let generateBeta = (): string =>
+  `beta(${generatePositiveInt()}, ${generatePositiveInt()})`;
+
+export let generateLognormal = (): string =>
+  `lognormal(${generateInt()}, ${generatePositiveInt()})`;
+
+export let generateExponential = (): string =>
+  `exponential(${generatePositiveInt()})`;
+
+export let generateUniform = (): string => {
+  let a = generateInt();
+  let b = generateIntMin(a + 1);
+  return `uniform(${a}, ${b})`;
+};
+export let generateCauchy = (): string => {
+  return `cauchy(${generateInt()}, ${generatePositiveInt()})`;
+};
+
+export let generateTriangular = (): string => {
+  let a = generateInt();
+  let b = generateIntMin(a + 1);
+  let c = generateIntMin(b + 1);
```

Comment:
what's with all the ``a/b+1``? Any reason not to use a random number there?

Reply:
I use +1 because the random number generator is inclusive of the minimum. Which would make an error if it's the same as a or b.